bin/list-all

```diff
@@ -3,11 +3,7 @@
 set -eo pipefail
 
 cmd="curl --silent --location"
```

Comment:
Since we're making changes around this area of the code, might I suggest we use [`curl -fsSL`](https://explainshell.com/explain?cmd=curl+-fsSL+example.org#)?